File: Plugins/Aspose_Pdf_Java_for_PHP/src/Aspose/Pdf/WorkingWithDocumentConversion/SvgToPdf.php
<?php

namespace Aspose\Pdf\WorkingWithDocumentConversion;

use com\aspose\pdf\Document as Document;
use com\aspose\pdf\SvgLoadOptions as SvgLoadOptions;

class SvgToPdf {

    public static function run($dataDir=null)
    {
        # Instantiate LoadOption object using SVG load option
        $options = new SvgLoadOptions();

        # Create document object
        $pdf = new Document($dataDir . 'Example.svg', $options);

        # Save the output to XLS format
        $pdf->save($dataDir . "SVG.pdf");

        print "Document has been converted successfully";
    }
}

File: Plugins/Aspose-Pdf-Java-for-Jython/asposepdf/WorkingWithDocumentConversion/SvgToPdf.py
from asposepdf import Settings
from com.aspose.pdf import Document
from com.aspose.pdf import SvgLoadOptions

class SvgToPdf:

    def __init__(self):
        dataDir = Settings.dataDir + 'WorkingWithDocumentConversion/SvgToPdf/'
        
        # Instantiate LoadOption object using SVG load option
        options = SvgLoadOptions()

        # Create document object
        pdf = Document(dataDir + 'Example.svg', options)

        # Save the output to XLS format
        pdf.save(dataDir + "SVG.pdf")

        print "Document has been converted successfully"
 
if __name__ == '__main__':        
    SvgToPdf()